### ISSUE TITLE
Enemies not dying after getting shot

### ISSUE THREAD

**Issue by SomethingSimpl** [LordGolias/antistasi#23]
Hi,

Iam back!

Tried the latest commit. Some enemies are stuck in deathless limbo after successfully shooting them.
They are down but can move their heads/torsos but never wake up. After they get into this state nothing is harming them further but they appear to never wake up.

**Comment by LordGolias** [comment]
Is this using ACE or without ACE?

**Comment by SomethingSimpl** [comment]
Oh man, thats what I get for not reading through the issue reports properly :)

**Comment by SomethingSimpl** [comment]
Hm, but seeing the ACE issue report there doesnt seem to be any proper solution other than not using the preventinstadeath setting, right?

Or do you believe that this is a recent bug introduced in latest releases of ACE?

**Comment by LordGolias** [comment]
Could you describe what `preventinstadeath` is about? I hit my head very hard reading the [ACE handleDamage](https://github.com/acemod/ACE3/blob/master/addons/concertina_wire/functions/fnc_handleDamage.sqf) and was not able to understand why the units are not dying when unconscious.

**Comment by SomethingSimpl** [comment]
Well as far as I understand, its a module setting within ACE that basically moves the unit to unconscious state after receiving fatal damage. You can check the option out in editor after placing "Medical Settings" module.

Couldnt this be connected to the new "Respawn" and "AI Revive" script you put in? The one that pops up right after you die?

**Comment by LordGolias** [comment]
Thanks for the explanation. 

> Couldnt this be connected to the new "Respawn" and "AI Revive" script you put in? 

It can. I have to double check it.

**Comment by SomethingSimpl** [comment]
Oooh I got it.

The way it works is this:

Default setting of the medical module is: AI has a 50/50 chance of falling into unconscious state.
You shoot at 2 AIs. 1 = dead, 1 = unconscious.
The unconscious AI gets picked up by ACE_fnc_setUnconscious.sqf and ACE_fncisUnconscious.sqf that in turn gets turned into AS_fnc_isUnconscious.sqf that the revive script works with, considers it viable to use the Revive script on the AI making it unkillable on the timer.

TL;DR
We need to set the AI unconsciousness setting from the Medical Settings module from `50/50 `to `Disabled` therefore making it so that AI isnt considered for the revive script at all.

**Comment by SomethingSimpl** [comment]
Adding this into aceSettings.hpp:
```
class ace_medical_enableUnconsciousnessAI {
        title = "Unconscious AI";
        ACE_setting = 1;
        values[] = {"Disabled", "50/50", "Enabled"};
        texts[] = {"Disabled", "50/50", "Enabled"};
        default = 0;
        typeName = "BOOL";
        force = 1;
};
```
Should fix the issue. Tested it briefly.
